contracts/VerifierRegistry.sol

```diff
@@ -93,15 +103,23 @@ contract VerifierRegistry is Ownable {
     return true;
   }
 
-  function update(string _location) public {
+  function update(string _name, string _location) public {
     Verifier storage verifier = verifiers[msg.sender];
 
     require(verifier.id != address(0), "verifier do not exists");
 
+    bytes32 hash = hashName(_name);
+    bytes32 oldHash = hashName(verifier.name);
+    require(hash == oldHash || !uniqueNames[hash], "specified name is not available");
```

Comment:
should't it be:
```suggestion
    require(hash != oldHash && !uniqueNames[hash], "specified name is not available");
```

or maybe even better:
```suggestion
    require(hash != oldHash, "name is the same");
    require(!uniqueNames[hash], "specified name is not available");
```

Reply:
the checking is correct. we can update with the same name.

Reply:
oh... but why we want to allow update with same name? it will do nothing but use gas.